packages/cli/src/commands/diff.ts

```diff
@@ -122,16 +119,19 @@ export const command = (
 
 const diffBuilder = createCommandBuilder({
   options: {
-    command: 'diff <toEnv> [filters..]',
+    command: 'diff <from-env> <toEnv> [elm-selectors..]',
```

Comment:
nit: align the positional names

```suggestion
    command: 'diff <from-env> <to-env> [elm-selectors..]',
```